diff -ur metrics-reporter-config-3.0.2/reporter-config3/src/main/java/com/addthis/metrics3/reporter/config/ReporterConfig.java metrics-reporter-config-3.0.2.new/reporter-config3/src/main/java/com/addthis/metrics3/reporter/config/ReporterConfig.java
--- metrics-reporter-config-3.0.2/reporter-config3/src/main/java/com/addthis/metrics3/reporter/config/ReporterConfig.java	2016-08-17 21:20:22.000000000 +0200
+++ metrics-reporter-config-3.0.2.new/reporter-config3/src/main/java/com/addthis/metrics3/reporter/config/ReporterConfig.java	2016-08-23 10:14:58.894233336 +0200
@@ -44,13 +44,7 @@
     @Valid
     private List<GraphiteReporterConfig> graphite;
     @Valid
-    private List<InfluxDBReporterConfig> influxdb;
-    @Valid
     private List<RiemannReporterConfig> riemann;
-    @Valid
-    private List<StatsDReporterConfig> statsd;
-    @Valid
-    private List<ZabbixReporterConfig> zabbix;
 
     public List<ConsoleReporterConfig> getConsole()
     {
@@ -92,15 +86,6 @@
         this.graphite = graphite;
     }
 
-    public List<InfluxDBReporterConfig> getInfluxdb()
-    {
-        return influxdb;
-    }
-
-    public void setInfluxdb(List<InfluxDBReporterConfig> influxdb)
-    {
-        this.influxdb = influxdb;
-    }
 
     public List<RiemannReporterConfig> getRiemann()
     {
@@ -112,26 +97,6 @@
         this.riemann = riemann;
     }
 
-    public List<StatsDReporterConfig> getStatsd()
-    {
-        return statsd;
-    }
-
-    public void setStatsd(List<StatsDReporterConfig> statsd)
-    {
-        this.statsd = statsd;
-    }
-
-    public List<ZabbixReporterConfig> getZabbix()
-    {
-        return zabbix;
-    }
-
-    public void setZabbix(List<ZabbixReporterConfig> zabbix)
-    {
-        this.zabbix = zabbix;
-    }
-
     public boolean enableConsole(MetricRegistry registry)
     {
         boolean failures = false;
@@ -204,24 +169,6 @@
         return !failures;
     }
 
-    public boolean enableInfluxdb(MetricRegistry registry)
-    {
-        boolean failures = false;
-        if (influxdb == null)
-        {
-            log.debug("Asked to enable influx, but it was not configured");
-            return false;
-        }
-        for (InfluxDBReporterConfig influxConfig : influxdb)
-        {
-            if (!influxConfig.enable(registry))
-            {
-                failures = true;
-            }
-        }
-        return !failures;
-    }
-
     public boolean enableRiemann(MetricRegistry registry)
     {
         boolean failures = false;
@@ -240,42 +187,6 @@
         return !failures;
     }
 
-    public boolean enableStatsd(MetricRegistry registry)
-    {
-        boolean failures = false;
-        if (statsd == null)
-        {
-            log.debug("Asked to enable statsd, but it was not configured");
-            return false;
-        }
-        for (StatsDReporterConfig statsdConfig : statsd)
-        {
-            if (!statsdConfig.enable(registry))
-            {
-                failures = true;
-            }
-        }
-        return !failures;
-    }
-
-    public boolean enableZabbix(MetricRegistry registry)
-    {
-        boolean failures = false;
-        if (zabbix == null)
-        {
-            log.debug("Asked to enable zabbix, but it was not configured");
-            return false;
-        }
-        for (ZabbixReporterConfig zabbixConfig : zabbix)
-        {
-            if (!zabbixConfig.enable(registry))
-            {
-                failures = true;
-            }
-        }
-        return !failures;
-    }
-
     public boolean enableAll(MetricRegistry registry)
     {
         boolean enabled = false;
@@ -295,22 +206,10 @@
         {
             enabled = true;
         }
-        if (influxdb != null && enableInfluxdb(registry))
-        {
-            enabled = true;
-        }
         if (riemann != null && enableRiemann(registry))
         {
             enabled = true;
         }
-        if (statsd != null && enableStatsd(registry))
-        {
-            enabled = true;
-        }
-        if (zabbix != null && enableZabbix(registry))
-        {
-            enabled = true;
-        }
         if (!enabled)
         {
             log.warn("No reporters were succesfully enabled");
@@ -332,9 +231,7 @@
         report(csv);
         report(ganglia);
         report(graphite);
-        report(influxdb);
         report(riemann);
-        report(zabbix);
     }
 
     public static ReporterConfig loadFromFileAndValidate(String fileName) throws IOException
diff -ur metrics-reporter-config-3.0.2/reporter-config3/src/test/java/com/addthis/metrics3/reporter/config/sample/SampleTest.java metrics-reporter-config-3.0.2.new/reporter-config3/src/test/java/com/addthis/metrics3/reporter/config/sample/SampleTest.java
--- metrics-reporter-config-3.0.2/reporter-config3/src/test/java/com/addthis/metrics3/reporter/config/sample/SampleTest.java	2016-08-17 21:20:22.000000000 +0200
+++ metrics-reporter-config-3.0.2.new/reporter-config3/src/test/java/com/addthis/metrics3/reporter/config/sample/SampleTest.java	2016-08-23 10:30:31.703429151 +0200
@@ -119,37 +119,6 @@
     }
 
     @Test
-    public void sampleInfluxDB() throws Exception
-    {
-        ReporterConfig config = ReporterConfig.loadFromFile("src/test/resources/sample/influxdb.yaml");
-        System.out.println(yaml.dump(config));
-        log.info("Sample InfluxDB");
-        runLoop(config);
-    }
-
-    @Test
-    public void sampleStatsD() throws Exception
-    {
-        ReporterConfig config = ReporterConfig.loadFromFile("src/test/resources/sample/statsd.yaml");
-        System.out.println(yaml.dump(config));
-        log.info("Sample StatsD");
-        assertNotNull(config.getStatsd());
-        assertEquals(1, config.getStatsd().size());
-        runLoop(config);
-    }
-
-    @Test
-    public void sampleStatsDMulti() throws Exception
-    {
-        ReporterConfig config = ReporterConfig.loadFromFile("src/test/resources/sample/statsd-multi.yaml");
-        System.out.println(yaml.dump(config));
-        log.info("StatsD Multi");
-        assertNotNull(config.getStatsd());
-        assertEquals(2, config.getStatsd().size());
-        runLoop(config);
-    }
-
-    @Test
     public void sampleMulti() throws Exception
     {
         ReporterConfig config = ReporterConfig.loadFromFile("src/test/resources/sample/multi.yaml");
